app/models/amt/AMTAssignmentTable.scala

```diff
@@ -66,6 +66,10 @@ object AMTAssignmentTable {
     q.update(completed)
   }
 
```

Comment:
This function should be placed before all the saving/updating functions (to match conventions of the rest of the codebase). It could also probably use a brief comment to say what it does more explicitly.